### PR TITLE
Timeline

### PR DESCRIPTION
**Pull request by viviehn**

### CODE REVIEW

File: _includes/annual-data.html [hunk 1]
```diff
@@ -0,0 +1,113 @@
+
+		
+		<script src="../assets/javascripts/Chart.js"></script>
+		<script src="../assets/javascripts/legend.js"></script>
+
+		<style>
+
+#fork {
```
Comment (jiangdaniel) [inline _includes/annual-data.html]: Is there a "fork" id anywhere on the page?

File: _includes/annual-data.html [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+
+		var ctx = document.getElementById("canvas").getContext("2d");
+pieData = new Chart(ctx).Bar(pieAnnualData, {
+			responsive : true 
+		});
+	
+
+legend(document.getElementById("legend"), pieAnnualData, pieData);
+show()
```
Comment (jiangdaniel) [inline _includes/annual-data.html]: Minor thing: seems to be missing a semicolon

File: _includes/annual-data.html [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+				label: "Mentors",
+				fillColor : "rgba(151,187,205,0.5)",
+				strokeColor : "rgba(151,187,205,0.8)",
+				highlightFill : "rgba(151,187,205,0.75)",
+				highlightStroke : "rgba(151,187,205,1)",
+				data : [35, 90, 107, 86, 76]
+			}
+		]
+
+	}
```
Comment (jiangdaniel) [inline _includes/annual-data.html]: Minor thing: seems to be missing a semicolon

File: compete/rc-fall.html [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: default
 hero-image: url(../assets/images/stock-images/robotics-competition.jpg)
-hero-message: Fall Competition
+hero-message: PIE Fall Competition
```
Comment (jiangdaniel) [inline compete/rc-fall.html]: Is this meant to be "PiE"?

File: compete/rc-spring.html [hunk 5]
```diff
@@ -1,40 +1,163 @@
 ---
 layout: default
 hero-image: url(../assets/images/stock-images/robotics-competition.jpg)
-hero-message: Spring Competition
+hero-message: PIE Robotics Competition
```
Comment (jiangdaniel) [inline compete/rc-spring.html]: Is this meant to be "PiE"?

File: _includes/annual-data.html (hunk 1 above)
Comment (viviehn) [inline _includes/annual-data.html]: fixed by cd61b58

File: _includes/annual-data.html (hunk 2 above)
Comment (viviehn) [inline _includes/annual-data.html]: fixed by cd61b58

File: _includes/annual-data.html (hunk 3 above)
Comment (viviehn) [inline _includes/annual-data.html]: fixed by cd61b58

File: compete/rc-fall.html (hunk 4 above)
Comment (viviehn) [inline compete/rc-fall.html]: fixed by 94c5a88

File: compete/rc-spring.html (hunk 5 above)
Comment (viviehn) [inline compete/rc-spring.html]: fixed by 94c5a88

**Pull request merged**